pkg/ccl/backupccl/restore_job.go

```diff
@@ -692,6 +694,32 @@ func backedUpDescriptorWithInProgressImportInto(
 	return true, nil
 }
 
+// validateRestoredDescriptors returns an error if any descriptor in the cluster is invalid.
+func validateRestoredDescriptors(ctx context.Context, execConfig *sql.ExecutorConfig) error {
+	allDescs, err := backupresolver.LoadAllDescs(ctx, execConfig, hlc.Timestamp{WallTime: timeutil.Now().
+		UnixNano()})
+	if err != nil {
+		return err
+	}
+	targetDescs := make([]catalog.Descriptor, 0)
+	for _, desc := range allDescs {
+		if desc.Dropped() {
+			// For some reason, ValidateDescriptors fails on the temp system table
+			// when they're in the dropped stage.
```

Comment:
Do you have an example of the validation failure?

Reply:
If I remove this skip line, all cluster restores fail with the following error msg on each temp system table  "no matching name info in draining names of dropped relation"

Reply:
@fqazi is it surprising that this validation check would fail on dropping descriptors?

Reply:
The fact its temp table makes in interesting, do you know the exact error? Was it related to namespace entries? Wondering if its a validation bug